website/client-old/js/services/userServices.js

```diff
@@ -318,9 +318,8 @@ angular.module('habitrpg')
         },
 
         addTag: function(data) {
-          user.ops.addTag(data);
           Tags.createTag(data.body).then(function(res) {
-            sync();
+            user.ops.addTag({body: res.data.data});
```

Comment:
Would this be better for creating tags? This way they are not available to use until the server responds with the correct id.